public/js/Lineplot.js

```diff
@@ -199,14 +212,12 @@ Lineplot.prototype.keypressUpdate = function(keyCode) {
 
     if(self.activeFeature) {
         if(keyCode === 39 && self.activeFeature < this.featureLength) {
-            this.activeFeature++;
-            self.histogram.drawHistogram(self.activeFeature-1);
-            self.updateActiveLine(self.activeFeature);
+            self.activeFeature++;
+            $('body').trigger('linePlotUpdate', self.activeFeature);
         }
         else if(keyCode === 37 && self.activeFeature > 1) {
```

Comment:
Ooh, can you create some constants for these, say LEFTARROW and RIGHTARROW? It's both more readable and less brittle.